### PR TITLE
fix: ui improvements

### PR DESCRIPTION
**Pull request by qiweiii**
fix https://github.com/Magickbase/godwoken_explorer/issues/1268

Fixed some inconsistency from design, mainly make hash display become responsive in mobile screen

image

image

### CODE REVIEW

File: components/SmartContractInfo/styles.module.scss [hunk 1]
```diff
@@ -18,3 +18,15 @@
 .item-dd-cls {
```
[the remark this answers is not in the document]
Comment (qiweiii) [reply components/SmartContractInfo/styles.module.scss:18]: used, but redundant, can be removed

File: cypress/integration/tokens/index.spec.ts [hunk 2]
```diff
@@ -59,15 +59,15 @@ context('Tokens List Pages Tests', () => {
                 expect(link).to.has.css('pointer-events', 'none')
               })
           })
-          it('should navigate to next page on click next arrow', () => {
+          it.skip('should navigate to next page on click next arrow', () => {
```
[the remark this answers is not in the document]
Comment (qiweiii) [reply cypress/integration/tokens/index.spec.ts:62]: I forgot to remove the skip

**Review by JeffreyMa597 — APPROVED** [review]

File: components/ResponsiveHash/index.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  copyAlertText?: string
+}) => {
+  const [isMobile, setIsMobile] = useState(false)
+  const [modifiedLabel, setModifiedLabel] = useState(label)
+
+  useEffect(() => {
+    if (window && window.matchMedia('(max-width: 1024px)').matches) {
+      setIsMobile(true)
+    }
+  }, [])
```
Comment (Keith-CY) [inline components/ResponsiveHash/index.tsx:37]: `useMemo` could be used if the value is computed on mount, but `isMobile` won't be updated on resize if it's only set on mount

Comment (qiweiii) [reply components/ResponsiveHash/index.tsx:37]: maybe make it update on resize is better, I can add a resize event listener

**Comment by FrederLu** [comment]
When `tokenId` is longer, the display format needs to be adjusted. [Look at points 6 and 7 for the style](https://github.com/Magickbase/godwoken_explorer/issues/1268#issue-1525173143).
image
image

**Review by FrederLu — APPROVED** [review]

**Pull request merged**